custom/icds_reports/tests/test_inactive_dashboard_users.py

```diff
@@ -41,10 +41,11 @@ def test_get_inactive_users_data_added(self):
                 csv_reader = csv.reader(items_file)
                 data = list(csv_reader)
                 sorted(data, key=lambda x: x[0])
-                self.assertEqual([['Username', 'Location', 'State'],
-                                  ['234.testing@icds-cas.commcare.org', '', ''],
-                                  ['123.testing@icds-cas.commcare.org', '', '']
-                                  ], data)
+                self.assertEqual(['Username', 'Location', 'State'], data[0])
+                self.assertCountEqual([
```

Comment:
this seems like a terrible name https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertCountEqual It hought it was only testing the number of items. not "Test that sequence first contains the same elements as second, regardless of their order"

Reply:
I think ```assertItemsEqual``` is clearer but that we should go with the Python 3 rename after the upgrade and get rid of ```assertItemsEqual``` from our tests.  I think it's easier for new developers if we follow standard Python practices instead of trying to do our own thing, regardless of my own feelings about the rename.

Reply:
+1 I agree with going with assertCountEqual. I was just complaining about python's choices. not yours

Reply:
hahaha